auctions/README.md

```diff
@@ -0,0 +1,183 @@
+# Auctions
+
+## Firestore collections:
+- `auctions`
+- `biddingLog`
+
+## Auctions Metadata object
+
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <example: neelam>,
+  'quantity': <example: 10>,
+  'number_of_bidders': <example: 100>,
+  'highest_bidder': <userId>,
+  'highest_bid': <example: 200 dinero | default: initial_price>,
+  'start_time': <unix-timestamp>,
+  'end_time': <unix-timestamp>
+}
+```
+
+## Auction Details Object
+
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
```

Comment:
If this is the response object, shouldnt we send a username of the seller?
@ankushdharkar meant that we should be storing userId in the DB, but show the username in response.

Can you also add the DB schema being used?

Reply:
Yeah, will change to `username`.
Okay, will do that.